src/pounder/mod.rs

```diff
@@ -32,34 +32,109 @@ const ATT_LE3_PIN: u8 = 8 + 3;
 
 pub struct QspiInterface {
     pub qspi: hal::qspi::Qspi,
+    mode: ad9959::Mode,
+}
+
+impl QspiInterface {
+    pub fn new(mut qspi: hal::qspi::Qspi) -> Result<Self, Error> {
+        qspi.configure_mode(hal::qspi::QspiMode::FourBit).map_err(|_| Error::Qspi)?;
+        Ok(Self { qspi: qspi, mode: ad9959::Mode::SingleBitTwoWire })
```

Comment:
This apparent contradiction and the lack of `qspi.configure_mode()` anywhere else should probably get a comment.